src/rtpghi~.c

```diff
@@ -18,6 +34,7 @@ typedef struct _rtpghi_tilde {
 	t_symbol *window_type_pd;
 	phaseret_rtpghi_state_s* sta_pd;
 	ltfat_complex_s *c;
+	ltfat_int blocksize;
```

Comment:
here we still have two single-precision members (which won't work with `PD_FLOATSIZE=64`).